library/tiqr/Tiqr/Service.php

```diff
@@ -201,9 +201,7 @@ public function __construct(LoggerInterface $logger, $options=array(), $version
             $type = $options["statestorage"]["type"];
             $storageOptions = $options["statestorage"];
         } else {
-            $this->logger->info('Falling back to file state storage');
-            $type = "file";
-            $storageOptions = array();
+            throw new RuntimeException('No state storage configuration is configured, please provide one');
```

Comment:
Boy scout rule: Configuring the state storage explicitly is better than a magic fallback in my opinion!